Cython/Utility/CommonStructures.c

```diff
@@ -102,9 +102,15 @@ static PyTypeObject *__Pyx_FetchCommonTypeFromSpec(PyTypeObject *metaclass, PyOb
         goto bad;
     }
 
-    // We pass the ABI module reference to avoid keeping the user module alive by foreign type usages.
+#if !CYTHON_USE_MODULE_STATE
+    // Without module-state, pass the ABI module reference to avoid keeping the user module alive by foreign type usages.
+    // With module-state it's important to keep the user module alive though.
     CYTHON_UNUSED_VAR(module);
```

Comment:
With your change below, `module` is used either way. I think you can just delete this line.